packages/jumpstarter-driver-android/jumpstarter_driver_android/driver/adb.py

```
@@ -0,0 +1,105 @@
+                stderr=subprocess.PIPE,
+                text=True,
+                env={"ANDROID_ADB_SERVER_PORT": str(self.port), **dict(os.environ)},
+            )
+            self._print_output(result.stdout)
+            self._print_output(result.stderr, error=True)
+            self.logger.info(f"ADB server stopped on port {self.port}")
+        except subprocess.CalledProcessError as e:
+            self._print_output(e.stdout)
+            self._print_output(e.stderr, error=True)
+            self.logger.error(f"Failed to stop ADB server: {e}")
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)